internal/controller/bucket/bucket.go

```diff
@@ -217,10 +225,8 @@ func (c *external) Create(ctx context.Context, mg resource.Managed) (managed.Ext
 		return managed.ExternalCreation{}, errors.New(errNotBucket)
 	}
 
-	backends := newBackendStatuses()
-
 	bucket.Status.SetConditions(xpv1.Creating())
-	defer setBucketStatus(bucket, backends.getBackendStatuses())
+	defer setBucketStatus(bucket, c.backendStatuses.getBackendStatuses()) // backends.getBackendStatuses())
```

Comment:
I imagine leaving this comment here was accidental?

Reply:
Yep, thanks :+1: